config/clusterlifecycle.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterlifecycles.kpack.io
+spec:
+  group: kpack.io
+  versions:
+  - name: v1alpha1
```

Comment:
Since there is no existing resource, we should only have v1alpha2 here

Reply:
Updated in aea38633